src/NMoneys/Currency.cs

```diff
@@ -299,10 +312,14 @@ private void setAllFields(CurrencyIsoCode isoCode, string englishName, string na
 		/// Testing currency
 		/// </summary>
 		public static readonly Currency Test;
+        /// <summary>
+		/// Currency with customizable format
+		/// </summary>
+		public static readonly Currency Zzz;
 
-		/*private static readonly ThreadSafeCache<string, Currency> _byIsoSymbol;
+        /*private static readonly ThreadSafeCache<string, Currency> _byIsoSymbol;
 		private static readonly ThreadSafeCache<CurrencyIsoCode, Currency> _byIsoCode;*/
-		private static readonly CurrencyCache _cache;
+        private static readonly CurrencyCache _cache;

```

Comment:
We should fix all of this reformatting stuff, I'd prefer to keep the changes to only the lines we need to change.